girder-dandi-archive/tests/test_rest.py

```diff
@@ -44,38 +44,38 @@ def testCreateDandiset(server, draftsFolders, user):
         method="POST",
         user=user,
         params={
-            "name": "test dandiset 0 name",
-            "description": "test dandiset 0 description",
+            "name": "test dandiset 1 name",
+            "description": "test dandiset 1 description",
         },
     )
     assertStatusOk(resp)
-    test_dandiset_0_folder = resp.json
-    test_dandiset_0_folder_meta = test_dandiset_0_folder["meta"]["dandiset"]
+    test_dandiset_1_folder = resp.json
+    test_dandiset_1_folder_meta = test_dandiset_1_folder["meta"]["dandiset"]
 
-    assert drafts_collection_id == test_dandiset_0_folder["parentId"]
-    assert "000000" == test_dandiset_0_folder["name"]
-    assert "000000" == test_dandiset_0_folder_meta["identifier"]
-    assert "test dandiset 0 name" == test_dandiset_0_folder_meta["name"]
-    assert "test dandiset 0 description" == test_dandiset_0_folder_meta["description"]
+    assert drafts_collection_id == test_dandiset_1_folder["parentId"]
+    assert "000001" == test_dandiset_1_folder["name"]
```

Comment:
What are your thoughts on having a test that just tests the counter behavior?

Reply:
I'd say that it's already covered by `testCreateDandiset`. Counter behavior is a part of the dandiset creation process. A counter specific test would be exactly the same as `testCreateDandiset`, but with only the ID assert.